internal/command/postgres/failover.go

```diff
@@ -128,3 +141,170 @@ func runFailover(ctx context.Context) (err error) {
 	fmt.Fprintf(io.Out, "Failover complete\n")
 	return
 }
+
+func flexFailover(ctx context.Context, machines []*api.Machine, app *api.AppCompact) error {
+	io := iostreams.FromContext(ctx)
+
+	leader, err := pickLeader(ctx, machines)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(io.Out, "Performing a failover\n")
+
+	primary_region := ""
+	if len(machines) > 0 {
```

Comment:
We should abort if there's fewer than 3 running machines within the primary region, otherwise quorum can't be met and the failover will fail.